analyzers/its/sources/RazorSample/RazorSample/Pages/Shared/_Layout.cshtml.css

```diff
@@ -0,0 +1,48 @@
+﻿/* Please see documentation at https://docs.microsoft.com/aspnet/core/client-side/bundling-and-minification
```

Comment:
This file can be removed.

Reply:
Those files, together with `razor.css` are not showing up in VS solution explorer for some reason, thanks!